### PR TITLE
Add PR submission guide for Initiatives

### PR DESCRIPTION
**Pull request by bnb**
Added a guide to the `/local_initiatives/README.md` file for submitting a PR to add a local initiative. Feedback appreciated! 😊

(Also, accidentally committed this to `master` in 8b5bb663e73a7f1a95a785b13ca368e9faf712a2 - removed it in f780b4dc2250073ad954338299d1bb09908d98a1, as the intent was to submit a PR for review)

### CODE REVIEW

File: documents/local_initiatives/README.md [hunk 1]
```diff
@@ -11,4 +11,83 @@ You need three things to list a group in the directory.
...
+# Country OR Major Region
+## More Specific Region
+
+Location | Event | Website | Contact Point
+-------- | ----- | ------- | -------------
+Event Location  | Event Name | <Event Link> | Contact Method Name: Contact Method Address
+```
+
+Where...
+* `Country OR Major Region` is the name of the country or region events below would take place in. Some examples are `North America`, `Europe`, `Australia`, `Asia`, and so on.
```
Comment (vdeturckheim) [inline documents/local_initiatives/README.md:34]: Just a though I had while reading: maybe we can add a place for online initiatives without physical meetings.

**Comment by vdeturckheim** [comment]
lgtm :)

File: documents/local_initiatives/README.md (hunk 1 above)
Comment (bnb) [reply documents/local_initiatives/README.md:34]: Submit an issue or create a PR with that idea - _love_ it!

**Comment by bnb** [comment]
@homosaur Would love to get your feedback, as well 👍

**Comment by homosaur** [comment]
@bnb docs look good, but I noticed there's a "step 1" without any other steps, you might want to revise that

**Comment by danielkhan** [comment]
LGTM

**Review by vdeturckheim — APPROVED** [review]

**Comment by bnb** [comment]
Removed the Step 1 heading that @homosaur mentioned. Merging!

**Pull request merged**